conda.bzl

```diff
@@ -101,22 +109,32 @@ def _create_conda_build_file(rctx, executable):
 
 def _load_conda_impl(rctx):
     installer = _download_conda(rctx)
-    executable = _install_conda(rctx, installer)
-    _update_conda(rctx, executable)
-    _create_conda_build_file(rctx, executable)
+    conda_executable = _install_conda(rctx, installer)
+    if rctx.attr.install_mamba:
+        _install_mamba(rctx, conda_executable)
+    _update_conda(rctx, conda_executable)
+    _create_conda_build_file(rctx, conda_executable)
 
 load_conda_rule = repository_rule(
     _load_conda_impl,
     attrs = {
-        "conda_dir": attr.string(mandatory = True),
-        "version": attr.string(
+        "conda_dir": attr.string(mandatory=True),
```

Comment:
This change just reminded me about using `bazel` formatter: [`buildifier`](https://github.com/bazelbuild/buildtools/tree/master/buildifier). Maybe you already know about that, but please remember to format the files at the end.

Reply:
Thanks, I did not know about that.